collectivo/extensions/shifts/pages/shifts/my_shifts.vue

```diff
@@ -0,0 +1,114 @@
+<script setup lang="ts">
+import { readItems } from "@directus/sdk";
+import AssignmentCard from "~/components/AssignmentCard.vue";
+import {
+  getNextOccurrence,
+  isShiftDurationModelActive,
+} from "~/composables/shifts";
+import { DateTime } from "luxon";
+import { getUserLogs, getUserScore } from "~/composables/shift_logs";
+import { ShiftLogType } from "~/server/utils/ShiftLogType";
+
+setCollectivoTitle("My shifts");
+const directus = useDirectus();
+const user = useCollectivoUser();
+await user.value.load();
+
+const assignments: CollectivoAssignment[] = await directus.request(
```

Comment:
Can we have the database calls run in parallel? At the moment, if I read it right, they will run one after another before loading the component, right? Instead, we could remove the await keyword and add a v-if statement in the template to only load the component once all the relevant data is loaded.

Reply:
Good point, updated 👍